Competition2017/src/competition/subsystems/collector/commands/StopCollectorCommand.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.collector.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.collector.CollectorSubsystem;
+import xbot.common.command.BaseCommand;
+
+
+public class StopCollectorCommand extends BaseCommand {
+    final CollectorSubsystem collectorSystem;
+    private static Logger log = Logger.getLogger(CollectorSubsystem.class);
+    
+    @Inject
+    public StopCollectorCommand(CollectorSubsystem collectorSystem){
+        this.collectorSystem = collectorSystem;   
+        this.requires(collectorSystem);
+    }
+    @Override
+    public void initialize() {
+        log.info("Stopping Collector");
+        collectorSystem.stop();
```

Comment:
All commands should either run every execute or run just in initialize. Don't mix.